SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -32,6 +34,84 @@ void USpatialPlayerSpawner::Init(USpatialNetDriver* InNetDriver)
 	NetDriver = InNetDriver;
```

Comment:
Do we still need the NetDriver, or can we just take the Connection?

Reply:
`SpatialPlayerSpawner` touches a lot of stuff in the NetDriver - Connection, LoadBalanceStrategy, PackageMap... I'd like to avoid additional changes in this PR, though I think there is stuff to clean up.